kinto_http/session.py

```diff
@@ -106,8 +112,8 @@ def request(self, method, endpoint, data=None, permissions=None,
                 exception.response = resp
                 raise exception
 
-        resp.headers['User-Agent'] = 'kinto-http.py/2.1.1 requests/2.3.2 python/3.6.1'
-
+        resp.headers['User-Agent'] = 'kinto-http.py/'+kinto_http_version +' requests/'+ requests_version +' python/' + python_version
```

Comment:
Can you use [`.format()`](https://docs.python.org/2/library/stdtypes.html#str.format) for the string creation?